### PR TITLE
8320342: Use PassFailJFrame for TruncatedPopupMenuTest.java

### PR DESCRIPTION
**Pull request by rajamah**
Modified the test to use PassFailJFrame and open sourced it.
The test shows two windows, one for instructions and the other to perform testing on.
Tested locally and it works fine.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8320342](https://bugs.openjdk.org/browse/JDK-8320342): Use PassFailJFrame for TruncatedPopupMenuTest.java (**Bug** - P4)

### Reviewers
 * [Harshitha Onkar](https://openjdk.org/census#honkar) (@honkar-jdk - Committer) ⚠️ Review applies to [6c021d98](https://git.openjdk.org/jdk/pull/17641/files/6c021d984cf4423a4d6a3f0a69b38c3a7befe3be)
 * [Alexey Ivanov](https://openjdk.org/census#aivanov) (@aivanov-jdk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/17641/head:pull/17641` \
`$ git checkout pull/17641`

Update a local copy of the PR: \
`$ git checkout pull/17641` \
`$ git pull https://git.openjdk.org/jdk.git pull/17641/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 17641`

View PR using the GUI difftool: \
`$ git pr show -t 17641`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/17641.diff">https://git.openjdk.org/jdk/pull/17641.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/17641#issuecomment-1917998103)

### CODE REVIEW

**Review by honkar-jdk — APPROVED** [review]

File: test/jdk/java/awt/PopupMenu/TruncatedPopupMenuTest.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    public static void main(String[] args) throws Exception {
+        PassFailJFrame.builder()
+                      .instructions(INSTRUCTIONS)
+                      .rows(20)
+                      .columns(55)
+                      .testUI(TruncatedPopupMenuTest::createTestUI)
+                      .build()
+                      .awaitAndCheck();
+
+    }
```
Comment (aivanov-jdk) [inline test/jdk/java/awt/PopupMenu/TruncatedPopupMenuTest.java:64]:
```suggestion
                      .awaitAndCheck();
    }
```
I suggest removing this blank line in the end of the method.

File: test/jdk/java/awt/PopupMenu/TruncatedPopupMenuTest.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+ */
+
+import java.awt.Frame;
+import java.awt.Menu;
+import java.awt.MenuItem;
+import java.awt.PopupMenu;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+
+/* @test
```
Comment (aivanov-jdk) [inline test/jdk/java/awt/PopupMenu/TruncatedPopupMenuTest.java:31]:
```suggestion
/*
 * @test
```
Keep the first line of the comment empty; it's empty usually.

**Review by aivanov-jdk — APPROVED** [review]

**Pull request closed without merging**